### ISSUE TITLE
Unable to override arrays using the config override CLI 

### ISSUE THREAD

**Issue by chris-j-h** [Consensys-Incorporated/tessera#761]
There is an issue with applying overrides to arrays.  

For example consider a configfile containing 4 `serverConfigs` in the order `ThirdParty`, `ADMIN`, `Q2T` and `P2P`.  If you want to override the `serverAddress` for the `ThirdParty` and `ADMIN` servers you would run 
``` bash
tessera -configfile conf.json \
    --serverConfigs.serverAddress <new ThirdParty addr> \
    --serverConfigs.serverAddress <new ADMIN addr>
```
However, the resulting config contains only the `ThirdParty` and `ADMIN` servers (i.e. `Q2T` and `P2P` configs are lost) and the only field set in these server configs is the `serverAddress`.  

The config validation picks this up and so Tessera fails to start.

**Comment by ohbus** [comment]
Hey @chris-j-h and @prd-fox 

Is this issue resolved?

If not I would like to work on it. Thanks 😃

**Comment by chris-j-h** [comment]
Hi - the Tessera CLI was updated in #946 and included this fix as well as several others.  This was released in Tessera 0.10.3.  Usage is documented here https://docs.goquorum.com/en/latest/Privacy/Tessera/Configuration/Using%20CLI%20to%20override%20config%20v0.10.3/.

Apologies, this issue should have been closed.
